.phpcs.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset name="Project">
+    <description>Project Coding Standard</description>
+
+    <exclude-pattern>vendor/*</exclude-pattern>
+    <exclude-pattern>tests/*</exclude-pattern>
+
+    <rule ref="Yii2"/>
+    <rule ref="Generic.Commenting.DocComment">
+        <exclude name="Generic.Commenting.DocComment.MissingShort"/>
```

Comment:
we dont need custom code style ruleset for this extension. Yii2 default ruleset should be used instead